lib/ood_core/job/adapters/kubernetes/batch.rb

```diff
@@ -347,19 +344,18 @@ def set_gke_credentials(auth)
   end
 
   def set_context
-    cmd = "#{base_cmd} config set-context #{cluster}"
+    cmd = "#{kubectl_cmd} config set-context #{cluster}"
```

Comment:
Looks like we should use `#{context}` here, not cluster.  I know it's an existing bug, but may as well take care of it while we see it.

Reply:
Fixed. They are same thing, the `context` function just calls `cluster`, but if that ever changed this would be necessary.